app/services/streams/stream_transform_csv_row.service.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+/**
+ * @module StreamTransformCSVRowService
+ */
+
+const { Transform } = require('stream')
+const ConvertToCSVService = require('../convert_to_csv.service')
+
+/**
+ * Returns a Transform stream which turns an incoming array into a CSV row, ie. comma-separated, with each element
+ * wrapped in double quotes, and ending with a newline.
```

Comment:
Possibly a slight correction needed in the comment

```suggestion
 * Returns a Transform stream which turns an incoming array into a CSV row, ie. comma-separated, with text elements
 * wrapped in double-quotes, any quote chars escaped and ending with a newline.
```